doc/node.1

```diff
@@ -1,368 +1,352 @@
-.TH NODE 1 2016 Node.js Node.js
-
-.\" This is a man page comment.
-
-.\" Man page syntax (actually roff syntax) is somewhat obscure, but the
-.\" important part is is that .<letter> specifies <letter>'s syntax for that
-.\" line, and \f<letter> specifies it for the characters that follow.
-
-.\" .B   Bold line
-.\" .I   Italic line (Rendered as underlined text in terminals)
-.\" .BI  Alternating bold/italics without spaces between arguments.
-.\"      Use `\ ` to include an "unpaddable" (literal) space in the output.
-.\" .RI  Alternating roman/italic
-
-.\" See http://liw.fi/manpages/ for an overview, or http://www.troff.org/54.pdf
-.\" for detailed language reference.
-
-.\" Macro to display an underlined URL in bold
```

Comment:
I'd suggest retaining a informational header about the file format because many people are not familar with the arcane art of writing manpages. A few links to the format and maybe a example on how to lint and preview should be enough

Reply:
I somehow missed this comment. And yep, I'll definitely add a header with some links to informative resources... I held off doing this for the same reason as the commit message: I wasn't sure how it would be received (and I'm absurdly OCD about well-written commit messages, sometimes to a [point of being manic](https://github.com/Alhadis/language-apl/commit/7e5d47ae2a538d921541158468206afa9c2836d6)). :sweat_smile: